cleanlab/datalab/datalab.py

```diff
@@ -185,7 +185,8 @@ def find_issues(
             evenly distributed across the rows.
             The non-zero entries must be the distances between the corresponding examples. Self-distances must be omitted
             (i.e. the diagonal must be all zeros and the k nearest neighbors of each example must not include itself).
-
+            The data must be sorted in row-major order, and within each row, the values must be sorted in ascending order.
```

Comment:
I think this needs 1-2 sentences more clarification.  What does it mean the data must be sorted? I think readers would interpret "the data" as the original dataset whose ordering is out of our control.

Reply:
This explanation needed a code example and a more explicit data structure description.
I hope the latest addition make things clearer.